### PR TITLE
lint mdn links

### PR DESCRIPTION
**Pull request by PassionPenguin**
1. http => https for *mozilla.org
2. https://developer.mozilla.org/en-US => /en-US (1 file)

and 861 more found occurences of "/en-US/docs/Glossary"..should i replace some of them?

### CODE REVIEW

File: files/en-us/web/api/document/createelementns/index.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ elements from two different namespaces within a single document:
 
 ```xml
 <?xml version="1.0"?>
-<page xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
+<page xmlns="https://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
```
Comment (PassionPenguin) [inline files/en-us/web/api/document/createelementns/index.md:69]:
```suggestion
<page xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
```

File: files/en-us/web/api/element/removeattributens/index.md [hunk 2]
```diff
@@ -33,10 +33,10 @@ None ({{jsxref("undefined")}}).
 
 ```js
 // Given:
-//   <div id="div1" xmlns:special="http://www.mozilla.org/ns/specialspace"
+//   <div id="div1" xmlns:special="https://www.mozilla.org/ns/specialspace"
```
Comment (PassionPenguin) [inline files/en-us/web/api/element/removeattributens/index.md:36]:
```suggestion
//   <div id="div1" xmlns:special="http://www.mozilla.org/ns/specialspace"
```

File: files/en-us/web/api/element/removeattributens/index.md [hunk 3]
```diff
@@ -33,10 +33,10 @@ None ({{jsxref("undefined")}}).
...
 //     special:specialAlign="utterleft" width="200px" />
 d = document.getElementById("div1");
-d.removeAttributeNS("http://www.mozilla.org/ns/specialspace", "specialAlign");
+d.removeAttributeNS("https://www.mozilla.org/ns/specialspace", "specialAlign");
```
Comment (PassionPenguin) [inline files/en-us/web/api/element/removeattributens/index.md:39]:
```suggestion
d.removeAttributeNS("http://www.mozilla.org/ns/specialspace", "specialAlign");
```

File: files/en-us/web/api/element/setattributenodens/index.md [hunk 4]
```diff
@@ -31,11 +31,11 @@ The replaced attribute node, if any, returned by this function.
 ## Examples
 
 ```js
-// <div id="one" xmlns:myNS="http://www.mozilla.org/ns/specialspace"
+// <div id="one" xmlns:myNS="https://www.mozilla.org/ns/specialspace"
```
Comment (PassionPenguin) [inline files/en-us/web/api/element/setattributenodens/index.md:34]:
```suggestion
// <div id="one" xmlns:myNS="http://www.mozilla.org/ns/specialspace"
```

File: files/en-us/web/api/element/setattributenodens/index.md [hunk 5]
```diff
@@ -31,11 +31,11 @@ The replaced attribute node, if any, returned by this function.
...
 //            myNS:special-align="utterleft">one</div>
 // <div id="two">two</div>
 
-const myns = "http://www.mozilla.org/ns/specialspace";
+const myns = "https://www.mozilla.org/ns/specialspace";
```
Comment (PassionPenguin) [inline files/en-us/web/api/element/setattributenodens/index.md:38]:
```suggestion
const myns = "http://www.mozilla.org/ns/specialspace";
```

File: files/en-us/web/api/element/setattributens/index.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ None ({{jsxref("undefined")}}).
 ```js
 let d = document.getElementById("d1");
 d.setAttributeNS(
-  "http://www.mozilla.org/ns/specialspace",
+  "https://www.mozilla.org/ns/specialspace",
```
Comment (PassionPenguin) [inline files/en-us/web/api/element/setattributens/index.md:38]:
```suggestion
  "http://www.mozilla.org/ns/specialspace",
```

File: files/en-us/web/opensearch/index.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ The XML file that describes a search engine follows the basic template below. Se
 
 ```xml
 <OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
-                       xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+                       xmlns:moz="https://www.mozilla.org/2006/browser/search/">
```
Comment (PassionPenguin) [inline files/en-us/web/opensearch/index.md:23]:
```suggestion
                       xmlns:moz="http://www.mozilla.org/2006/browser/search/">
```

File: files/en-us/web/api/element/namespaceuri/index.md [hunk 8]
```diff
@@ -22,7 +22,7 @@ In this snippet, an element is being examined for its {{domxref("Element.localNa
 if (
   element.localName === "browser" &&
   element.namespaceURI ===
-    "http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
+    "https://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
```
Comment (PassionPenguin) [inline files/en-us/web/api/element/namespaceuri/index.md:25]:
```suggestion
    "http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
```

File: files/en-us/web/api/element/hasattributens/index.md [hunk 9]
```diff
@@ -33,7 +33,7 @@ A boolean.
 // Check that the attribute exists before you set a value
 const d = document.getElementById("div1");
 if (
-  d.hasAttributeNS("http://www.mozilla.org/ns/specialspace/", "special-align")
+  d.hasAttributeNS("https://www.mozilla.org/ns/specialspace/", "special-align")
```
Comment (PassionPenguin) [inline files/en-us/web/api/element/hasattributens/index.md:36]:
```suggestion
  d.hasAttributeNS("http://www.mozilla.org/ns/specialspace/", "special-align")
```

**Review by pepelsbey — APPROVED** [review]
Thank you!

File: files/en-us/glossary/camel_case/index.md [hunk 10]
```diff
@@ -18,6 +18,6 @@ Camel case is the most popular convention in JavaScript, Java, and various other
 
 ## See also
 
-- [Snake case](/en-US/docs/Glossary/Snake_case)
-- [Kebab case](/en-US/docs/Glossary/Kebab_case)
+- {{Glossary("Snake_case", "Snake Case")}}
+- {{Glossary("Kebab_case", "Kebab Case")}}
```
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/en-us/glossary/camel_case/index.md:22]: oops, you are right.

**Pull request merged**